assets/charts/components/rook/templates/serviceaccount.yaml

```diff
@@ -76,3 +76,10 @@ metadata:
   name: rook-csi-rbd-provisioner-sa
   namespace:  {{  .Release.Namespace }}
 {{ template "imagePullSecrets" . }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-ceph-admission-controller
+  namespace: {{  .Release.Namespace }}
+---
```

Comment:
This is from upstream right?

Reply:
Yep :-)

Reply:
All lokomotive specific changes have a comment on them.